ApolloExtra.lua

```diff
@@ -2308,103 +2332,134 @@ OlympusLootData.ApolloUpgrade = {
 					Speaker = "NPC_Ares_01", Portrait = "Portrait_Ares_Default_01",
-					Text = "Mortals are so bent on clinging to their lives, that many among them would gladly kill for it. The surge of emotion which they feel... one passion is no different from another, Lord Apollo, dear." },
+					Text = "The only form of art I recognize is war. Hmm. I have never seen anybody killing with words. Zagreus, take this instead." },
+			},
+			ApolloWithAphrodite01 =
+			{
+                Name = "ApolloWithAphrodite01",
+				PlayOnce = true,
+				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
+				HasTraitNameInRoom = "HyacinthTrait",
+				RequiredTextLines = { "ApolloGift01", "AphroditeGift01" },
+				{ Cue = "/VO/Apollo_0045",
+					StartSound = "/Leftovers/World Sounds/MapZoomInShort",
+					Text = "Aphrodite and I are here to help. We came to a conclusion. Without our help, you won't escape. Therefore... " },
+				{ Cue = "/VO/Aphrodite_0230",
+					PortraitExitWait = 0.35,
+					PreLineFunctionName = "BoonInteractPresentation", PreLineWait = 0.5,
+					Emote = "PortraitEmoteAffection",
+					EndSound = "/Leftovers/World Sounds/MapZoomInShort",
+					Speaker = "NPC_Aphrodite_01", Portrait = "Portrait_Aphrodite_Default_01",
+					Text = "This is for you, little godling! Oh my me! I can't wait until we are all here together!"
+				},
 			},
 			ApolloWithArtemis01 =
 			{
                 Name = "ApolloWithArtemis01",
 				PlayOnce = true,
 				PreEventFunctionName = "BoonInteractPresentation", PreEventFunctionArgs = { PickupWait = 1.0, },
-				HasTraitNameInRoom = "HeartsickCritDamageTrait",
+				HasTraitNameInRoom = "FamedDuetTrait",
 				RequiredTextLines = { "ApolloGift01", "ArtemisGift01" },
-				{ Cue = "/VO/Apollo_0185",
+				{ Cue = "/VO/Apollo_0046",
 					Emote = "PortraitEmoteAffection",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort",
-					Text = "Oh dearest! I have brought you keen-eyed Artemis, who, in her focus on the hunt, has not developed social graces suitable enough to come to you herself!" },
-				{ Cue = "/VO/Artemis_0239",
+					Text = "I brought my sister with me this time, Zagzag! With our help, you might stand a chance against your father. Go Zagzag!" },
+				{ Cue = "/VO/Artemis_0250",
 					PortraitExitWait = 0.35,
 					PreLineFunctionName = "BoonInteractPresentation", PreLineWait = 0.5,
 					StartSound = "/SFX/ArtemisBoonArrow",
 					EndSound = "/Leftovers/World Sounds/MapZoomInShort",
 					Emote = "PortraitEmoteDepressed",
 					Speaker = "NPC_Artemis_01", Portrait = "Portrait_Artemis_Default_01",
-					Text = "Social graces, she says! Going about like {#DialogueItalicFormat}that{#PreviousFormat}! I'm sure you know that I can find you when it's necessary, Zagreus. Though, how to get away from Apollo, I have no idea."
+					Text = "Are you seriously calling him that... I am so sorry Zagreus. I hope this can make him a little more to bearable..."
```

Comment:
"Are you seriously calling him that... I am so sorry, Zagreus. I hope this can make him a little more to bearable..."